Attendee.test.js

```diff
@@ -0,0 +1,20 @@
+import { Attendee } from './Attendee';
+
+test('Check for correct value of screenName', () => {
+    const attendee = new Attendee(); 
+    attendee.setScreenName("Jaz"); 
+    const name = attendee.getScreenName();
+    expect(name).toBe(attendee.screenName); 
```

Comment:
This is bad testing. You're testing the object with itself. This doesnt prove that it works. Also, you shouldnt be accessing "attendee.screenName" directly. It is a private property. Test the way you expect another engineer to use your API. When you write a test, you come up with an expected output, you then feed the object your testing some information, then you test that whatever that object returns matches the expected output.
Like this:

const attendee = new Attendee();
const expectedScreenName = "Jaz";
attendee.setScreenName(expectedScreenName);
const gotScreenName = attendee.getScreenName();
expect(gotScreenName).toBe(expectedScreenName);

Notice that this test is not circular, so to speak. It tests the output of the object with an expected output. 

Fix this on your other tests, too